### DISCUSSION TITLE
Divisão de Tarefas

### DISCUSSION THREAD

**Discussion by ikidoncc** [diego3g/rsxp-2023#12] [General]
Galera o que vocês acham de criar divisões entre os colaboradores, pequenos times.

Exemplo:
Time 01 - Revisão de PR
Time 02 - Desenvolvimento Front
Time 03 - Desenvolvimento Back
Time 04 - Analise de qualidade de Testes
Time 05 - Design UX / Design UI

E pode ir dividindo. A ideia é que funcione como um fluxo de empresa mesmo, assim podendo servir de experiência pra quem precisa, e de muita troca de conhecimento, já que tem alguns times pré definidos com quem você pode conversar e tirar dúvidas mais especificas, ou até outras possibilidades,

**Comment by brunoesposito** [comment]
@SystemOfLima seria uma boa opção boa para otimizar devio ao tempo.

**Comment by diego3g** [comment]
Prefiro não separar em times pré formados, mas nada impede que as pessoas organicamente se dividam em times. Isso porque acho que cria uma barreira a mais e um compromisso maior com as features do projeto. Estamos discutindo sobre organizações de tarefas também na discussão #17.

**Reply by carlosdavid0** [reply]
Acredito que a ideia não é categorizar os times separando os devs, é simplesmente sinalizar que existem esses times e que os devs se sintam a vontade para contribuir no time que haja melhor identificação.

**Comment by diego3g** [comment]
Galera criei um board simples de tarefas por enquanto via issues pra irmos nos organizando: #49 

Vou fechar essa discussão por enquanto pra termos mais espaço pra novas discussões 🤗
